x-pack/plugins/enterprise_search/public/applications/app_search/components/api_logs/utils.ts

```diff
@@ -19,3 +19,11 @@ export const getStatusColor = (status: number) => {
   if (status >= 500) color = 'danger';
   return color;
 };
+
+export const safeJsonParseAndStringify = (json: string) => {
+  try {
+    return JSON.stringify(JSON.parse(json), null, 2);
+  } catch (e) {
```

Comment:
Also now that I actually view the code diff I just caught this lol

```suggestion
  } catch {
```